Source/Platform/DefaultOpenTK/Backend/Audio/NativeAudioSource.cs

```diff
@@ -320,6 +321,7 @@ private void PerformStreamingBegin()
 				}
 				else
 				{
+					this.strStopReq = StopRequest.EndOfStream;
 					break;
```

Comment:
I think this fix in its current form might introduce another problem: If we have a streamed, but very short audio sample, so that it fits completely into the streaming buffers at once, the initial buffer fill will `eof` and request stopping the source - so the audio will never be played, even though it would be perfectly fine to do so.

One possible solution might be to instead request stopping the stream only when triggering `eof` in the very first buffer already, so that we'd otherwise end up with a completely empty queue.